Total English Retranslation

glyphical opened the pull request:
If I'm doing this correctly, this branch contains a total English retranslation of the mod to give it perfect English in a consistent, professional style. In many cases, text was shortened and made easier to understand for end users. I realized I did the brackets wrong in the translation file I provided previously, so that has been fixed.

CoolLKKPS commented:
hello, i will take a look of this PR, hang on

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -238,118 +238,118 @@
   </data>
   <data name="msg_Map" xml:space="preserve">
-    <value>Detected player {player} using Mini-Map mod!</value>
+    <value>{player} was detected to have a mini-map mod. This can make finding loot much easier.</value>
```
yea this part is modified by me, but still u need to remove additional message to only keep original meaning

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -238,118 +238,118 @@
   </data>
   <data name="msg_Kick" xml:space="preserve">
-    <value>{player} was kicked!</value>
+    <value>{player} was banished!</value>
```
No, keep original meaning

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -238,118 +238,118 @@
   </data>
   <data name="msg_Enemy_SwitchToBehaviour" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster mechanism!</value>
+    <value>{player} was detected trying to alter a monster's behavior.</value>
```
im not sure this meaning, so for example the jester has 3 stages, if cheater try to change that will receive this message
im not sure how to say that correctly so i use mechanism word instead

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -238,118 +238,118 @@
   </data>
   <data name="msg_Enemy_ChangeOwnershipOfEnemy" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster behavior!</value>
+    <value>{player} was detected trying to control a monster.</value>
```
its not fully mean the player try to control the monster actually, cheater can make every monster to not targeting any player, that's not Control Company can do, its more than that

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -238,118 +238,118 @@
   </data>
   <data name="msg_ChatReal" xml:space="preserve">
-    <value>Detected player {player} tried to spoof {player2}'s name!</value>
+    <value>{player} tried to spoof {player2}'s name in chat.</value>
```
well i just removed "in chat" word, but its ok if really need

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_Turret" xml:space="preserve">
-    <value>Abnormal turret berserk mode detection</value>
+    <value>Detects when a player forces a turret to spin.</value>
```
again no turret spin

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_Shovel2" xml:space="preserve">
-    <value>Allow players to deal normal damage when empty-handed (network delays may cause players to be empty-handed)</value>
+    <value>Allow players to deal normal damage when empty-handed. Note: Network delays may cause players to be empty-handed.</value>
```
u should make this part more easier to read, and i didn't touch this part

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_ShipSetting" xml:space="preserve">
-    <value>Only the host can land the ship</value>
+    <value>Makes it so that only the host may land the ship.</value>
```
VERY ONLY the host can land the ship

glyphical replied:
I'm not sure I understand what the correction is here. The wording is what you said--it says that with this setting enabled, only the host can land the ship.

CoolLKKPS replied:
hmm changed to this is fine actually, ok

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_ShipLight" xml:space="preserve">
-    <value>Spamming light switch detection</value>
+    <value>Detects when a player spawns a light switch.</value>
```
this part is wrong

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_Nameless" xml:space="preserve">
-    <value>Player name detection (Nameless or Unknown)</value>
+    <value>Detects invalid or anomalous player names such as Nameless.</value>
```
such as Nameless or Unknown

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_Map" xml:space="preserve">
-    <value>Mini-Map mod detection</value>
+    <value>Detects players using a mini-map mod, displaying the radar on the screen at all times.</value>
```
remove additional part to keep original meaning

glyphical replied:
I feel like the additional information here is legitimately important. Not every player knows what a mini-map is or what it implies, and the UI has plenty of space for this info. I would like to recommend you keep this.

CoolLKKPS commented on AntiCheat/Lang/en_US.resx:
```diff
@@ -121,100 +121,100 @@
   </data>
   <data name="config_InfiniteAmmo" xml:space="preserve">
-    <value>Infinite ammo detection</value>
+    <value>Detects when a player has infinite ammo.</value>
```
u need to know, if the shotgun already loaded infinite ammo, normal player can also trigger that detection

glyphical replied:
I will address this and the rest of the problems you listed momentarily. Thank you.

CoolLKKPS reviewed (changes requested):
those issues and question u have to deal with

glyphical commented:
I addressed all the comments that I could. I do recommend re-integrating the minimap mod information, but otherwise it should be prepared for use now.

glyphical commented:
OK now it's ready, missed the lightswitch  issue.

CoolLKKPS commented:
we don't want to criticize another mod unless it does make the game environment more terrible, we more like to send feedback to mini-map mod author to fix those problem instead add opinions in the mod; And the host should have more experience, not the mod to teach them

glyphical commented:
I wouldn't say there was any criticism of the mod in my message--it simply said what minimaps do in completely neutral terms. Still I suppose I understand.

CoolLKKPS commented:
but still u can add opinions in config, just don't add it in message

glyphical commented:
Oh I understand now. I didn't want to keep the minimap info in the detection message, I only wanted to keep the info on what minimaps do in the config as I definitely understand removing the 'makes loot easier to find' thing. I already removed it though, it's no big deal--overall, you should be good to go with this now.

CoolLKKPS commented:
will notify mod author, and i may have to modify some readme-en, thx for ur translation!

chuxiaaaa commented:
Is everything ready?

glyphical commented:
It's prepared for merging as far as I can tell.

glyphical commented:
Just reviewed the text again and it's looking good. No typos that I saw.
